Legacy TypoScript condition globalVar causes parser error

Using legacy condition

```
# disable admin panel if editMode is set.
[globalVar = GP:editMode = 1]
  config.admPanel = 0
[global]
```

in setup.typoscript throws an error in our TYPO3-Logfile:

> [ERROR] request="c50649e6a38a9" component="TYPO3.CMS.Core.TypoScript.IncludeTree.Visitor.IncludeTreeConditionMatcherVisitor": TypoScript condition [globalVar = GP:editMode = 1] could not be parsed: Unexpected character "=" around position 10 for expression `globalVar = GP:editMode = 1`.- SyntaxError: Unexpected character "=" around position 10 for expression `globalVar = GP:editMode = 1`., in file /var/www/clients/clientx/webx/web/typo3_src-14.2.0/vendor/symfony/expression-language/Lexer.php:97

Possible solution:
```
[request && traverse(request.getQueryParams(),'editMode') == 1]
  config.admPanel = 0
[end]
```

TYPO3 14.2.0
visual_editor: 1.1.1

--- a/Configuration/TypoScript/setup.typoscript
+++ b/Configuration/TypoScript/setup.typoscript
@@ -6,7 +6,7 @@ tt_content.stdWrap.wrapContentElementsWithVeWrapper = 1
 # add parameter editMode to every link in edit mode.
 config.linkVars = editMode(1)
 
-# disable admin panel if editMode is set.
-[globalVar = GP:editMode = 1]
+# disable admin panel if editMode is set. this most likely dose not work as this setup.typoscript file is loaded as one of the first typoscript files
+[request && traverse(request.getQueryParams(),'editMode') == 1]
   config.admPanel = 0
-[global]
+[end]

